src/Payment/Service.php

```diff
@@ -28,7 +30,27 @@ public function create(
         string $currency = 'GBP',
         string $description = null,
         array $metadata = [],
-        string $customerUid = null
+        string $customerUid = null,
+        array $params = []
```

Comment:
is there a reason why we've gone to a params array here rather than more known/specific arguments?

Reply:
yes as there are a lot of parameters available to this API and it is unlikely that it will scale as we keep extending it.
Hence the wrapper method `reserve` as not all required parameters will be required yet we are calling the same API.